src/Playwright/Helpers/Driver.cs

```diff
@@ -37,7 +37,19 @@ internal static string GetExecutablePath()
             DirectoryInfo assemblyDirectory = new(AppContext.BaseDirectory);
             if (!assemblyDirectory.Exists || !File.Exists(Path.Combine(assemblyDirectory.FullName, "Microsoft.Playwright.dll")))
             {
+#if NETFRAMEWORK
```

Comment:
I am not sure if that works for a project targeting `netstandard2.0` only. Also I guess that this should be negated?

Reply:
Yup. This will always be false.

Reply:
Yeah, it's wrong, we should somehow check during the runtime if the application is `NETFRAMEWORK` since its always netstandard.

Reply:
Is the value different between .NET and .NET Framework? Maybe you can just always use CodeBase? 

You could check on runtime, but it is a bit annoying because the versions don't correspond exactly to the .NET versions. https://stackoverflow.com/questions/1826688/get-current-net-clr-version-at-runtime